dashboard/db/migrate/20160219021859_add_submitted_to_user_levels.rb

```diff
@@ -0,0 +1,14 @@
+class AddSubmittedToUserLevels < ActiveRecord::Migration
+  def change
+    add_column :user_levels, :submitted, :boolean
+
+    reversible do |change|
+      change.up do
+        execute('UPDATE user_levels SET submitted = true, best_result = 30 WHERE best_result = 1000')
```

Comment:
IMPORTANT: This means that the updated_at timestamp of this table will no longer be related to user activity. Before doing this migration, we should think about whether we want to add a "last_progress_at" or if the managed "created_at" column is sufficient for future analysis.

Note that since the activity table is unwieldy, these managed timestamps are often assumed to be when the user was working on the level.